New App: Clockodo 8.0.7

dwittenberg opened the pull request:
Add new App
 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/102219)

Commit:
New App: Clockodo 8.0.7

stephengillie commented:
Hi @dwittenberg,

The installation isn't unattended:
![InstallationVerification-x86-user-Locale_Undefined-ErrorScreenshot_20230411170330](https://user-images.githubusercontent.com/114952565/231261040-bc834930-6837-499e-8367-5a5a61f7602a.png)

Is there an installer switch to bypass this and have it install automatically?

dwittenberg committed:
Try fix installer

A working powershell command: .\clockodo-setup-8.0.7.exe /S /ALLUSERS=1 /D="C:\clockodo_clock"
